.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Can you remove this? It's not needed for this repo, and should probably be in your git `core.excludes` config.